### DISCUSSION TITLE
边上的标签是否支持为组件

### DISCUSSION THREAD

**Discussion by Xiaotong713986** [antvis/Graphin#464] [Q&A]
边上的标签不能支持为组件对吗？比如一个按钮：
![image](https://user-images.githubusercontent.com/23744239/205603584-9e13dba1-5ad1-4279-9a3e-5dcc8bed3c80.png)

**Comment by pomelo-nwu** [comment]
@Xiaotong713986 暂时不能，因为整个G6的设计都是基于Canvas的，如果引擎是 SVG，其实暴露出一个 foreignObject 即可渲染DOM。
